test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -12,5 +12,20 @@ public static string GetApplicationPath()
         {
             return Path.GetFullPath(Path.Combine("..", "..", "..", "..", "..", "ServerComparison.TestSites"));
         }
+
+        public static string GetConfigContent(ServerType serverType, string iisConfig, string nginxConfig)
```

Comment:
Isn't passing the name of the config files sort of defeats the purpose of this method? You should only pass the enum and get back the config... The names should be encapsulated.